alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunnerService.kt

```diff
@@ -142,20 +143,16 @@ object MonitorRunnerService : JobRunner, CoroutineScope, AbstractLifecycleCompon
             monitorCtx.maxActionableAlertCount = it
         }
 
-        monitorCtx.triggerMaxActions = ALERTING_TRIGGER_MAX_ACTIONS.get(monitorCtx.settings)
-        monitorCtx.triggerTotalMaxActions = ALERTING_TRIGGER_TOTAL_MAX_ACTIONS.get(monitorCtx.settings)
+        monitorCtx.maxActionsAcrossTriggers = MAX_ACTIONS_ACROSS_TRIGGERS.get(monitorCtx.settings)
+        monitorCtx.totalMaxActionsAcrossTriggers = TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS.get(monitorCtx.settings)
+        TriggersActionThresholdUtils.checkTriggerActionConfig(monitorCtx.maxActionsAcrossTriggers, monitorCtx.totalMaxActionsAcrossTriggers)
```

Comment:
This would consistently throw exceptions and prevent any monitor from running. This should only be done when the user tries to update the setting and prevent the check there or not even have the check.